client/src/components/ElementEditor/Content.js

```diff
@@ -26,9 +47,13 @@ class Content extends PureComponent {
             fileTitle={fileTitle}
           />
         }
-        {previewExpanded &&
+        {(this.state.elementFormLoaded || previewExpanded) &&
```

Comment:
For the sake of consistency it might be worth destructuring this state property. I haven't commented on this in the other PRs because it's more consistent in those cases not to do it =)

Reply:
Yeah, with all the other repos I guess it became habit :>